cpp/velox/jni/VeloxJniWrapper.cc

```diff
@@ -111,6 +111,8 @@ Java_io_glutenproject_vectorized_PlanEvaluatorJniWrapper_nativeValidateWithFailu
 
   // A query context used for function validation.
   velox::core::QueryCtx queryCtx;
+  std::unordered_map<std::string, std::string> configs{{velox::core::QueryConfig::kSparkPartitionId, "0"}};
```

Comment:
Do we need to pass this config for validator? Seems if lacked, it doesn't have any impact.

Reply:
Added, thanks